_projects/2022/100348375/100348375.Rmd

```diff
@@ -0,0 +1,981 @@
+
+sysfonts::font_add_google("Noto Serif")
+
+upper_plot <- upper_plot +
+    theme(axis.ticks.y = element_blank(),
+        axis.ticks.x = element_line(colour = "#c9c9c9", 
+                                    linewidth = c(0.3, 0.5, 0.5, 0.5, 0.8,
+                                                  0.5, 0.5, 0.5, 0.5, 0.4)),
+        axis.ticks.length.x = unit(0.2, "cm"),
+        axis.text.y = element_text(size = 10, family = "sans", 
+                                   hjust = 0, vjust = 0.5),
+        axis.text.x = element_text(size = 9, family = "sans", 
+                                   colour = "#a7a7a7",
+                                   face = c('plain', 'plain', 'plain', 
+                                            'plain', 'bold', 'plain', 
+                                            'plain', 'plain', 'plain', 'plain')),
+        plot.margin = margin(0, 1.7, 0, 0, "cm"))
+  
+upper_plot
+  
+```
+
+
+### DATA
+
+Now is the time to put the data on the chart. To plot the arrows from the 2000's 
+market share data to now I use "geom_segment". In this type of geom you have to 
+give it 4 aesthetics (x, xend, y, y-end). In addition, you can configure the type 
+of line (an arrow in this case) and the color, which in this graph depends on 
+whether the increase is positive or negative (variable **increase2** created 
+previously). Then, with the "scale_color_manual" function, the colors associated
+with each category of this variable are set. 
+Lastly, I've also added a thicker vertical line marking 50, using the 
+"geom_vline" function.
+
+```{r fig.height=7, fig.width=6, fig.showtext= TRUE}
+
+sysfonts::font_add_google("Noto Serif")
+
+upper_plot <- upper_plot +
+  geom_vline(xintercept = 50, color = "gray", linewidth = 0.8) + 
+  geom_segment(aes(x = market_share_2000, 
+                   xend = market_share_now, 
+                   y = industry_order, 
+                   yend = industry_order, 
+                   color = increase2), 
+               arrow = arrow(length = unit(0.15, "cm"), 
+                             type = "closed"), linewidth = 1) +
+  scale_color_manual(values = c("#575757", "#DB3B27"))
+
+upper_plot
+  
+```
+
+### ANNOTATIONS
+
+Finally, to finish the first graph, we must add the two annotations. To do this, 
+I have used the "annotate" function, using the x and y arguments to get the 
+proper final position within the plot area.
+
+```{r fig.height=7, fig.width=6, fig.showtext= TRUE}
+
+sysfonts::font_add_google("Noto Serif")
+
+upper_plot <- upper_plot +
+  annotate("text", x = 34.8, y = 23.9, label = "Early 2000s", 
+           size = 3, fontface = "bold") +
+  annotate("text", x = 86.2, y = 23.9, label = "Now", 
+           size = 3, fontface = "bold") +
+  annotate("text", x = 42, y = 23.4, label = "|", 
+           size = 3.2, fontface = "bold") +
+  annotate("text", x = 84, y = 23.4, label = "|", 
+           size = 3.2, fontface = "bold")
+
+upper_plot
+  
+```
+
+### All the code together:
```

Comment:
I suppose that this would be the last level 4 heading. Please do not use double column in titles.